integration-tests/constants/awsParameters.ts

```diff
@@ -1,9 +1,17 @@
 import { getEnv } from '../utils/helpers'
 
+export const ANALYSIS_BUCKET_NAME = getEnv('ANALYSIS_BUCKET_NAME')
```

Comment:
Can you also update the README/Confluence with the new env variables needed?

Reply:
This is already done.